homeassistant/components/binary_sensor/wink.py

```diff
@@ -1,45 +1,33 @@
 """
-Support for Wink sensors.
+Support for Wink binary sensors.
 
 For more details about this platform, please refer to the documentation at
-at https://home-assistant.io/components/sensor.wink/
+at https://home-assistant.io/components/binary_sensor.wink/
 """
-import logging
 import json
 
 from homeassistant.components.binary_sensor import BinarySensorDevice
 from homeassistant.components.sensor.wink import WinkDevice
-from homeassistant.const import CONF_ACCESS_TOKEN
 from homeassistant.helpers.entity import Entity
 from homeassistant.loader import get_component
 
-REQUIREMENTS = ['python-wink==0.7.14', 'pubnub==3.8.2']
+REQUIREMENTS = ['python-wink==0.7.15', 'pubnub==3.8.2']
 
 # These are the available sensors mapped to binary_sensor class
 SENSOR_TYPES = {
     "opened": "opening",
     "brightness": "light",
     "vibration": "vibration",
     "loudness": "sound",
-    "liquid_detected": "moisture"
+    "liquid_detected": "moisture",
+    "motion": "motion"
 }
 
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Setup the Wink binary sensor platform."""
     import pywink
 
-    if discovery_info is None:
```

Comment:
@balloob Great idea, then I'll only have to update the python-wink version in one place whenever there is an update. Pushing a change now, let me know if it looks okay.